packages/lib/server/getTeamMemberEmailFromCrm.ts

```diff
@@ -57,20 +72,14 @@ async function getAttributeRoutingConfig(
   if ("route" in data) {
     return data.route.attributeRoutingConfig ?? null;
   }
-  const { routingFormResponseId } = data;
+  const { routingFormResponseId, queuedFormResponseId } = data;
+  const routingFormResponseRepository = new RoutingFormResponseRepository(prisma);
 
-  const routingFormResponseQuery = await prisma.app_RoutingForms_FormResponse.findUnique({
-    where: {
-      id: routingFormResponseId,
-    },
-    include: {
-      form: {
-        select: {
-          routes: true,
-        },
-      },
-    },
-  });
+  const routingFormResponseQuery = routingFormResponseId
```

Comment:
The cause of the bug was if a `queuedResponseId` wasn't passed then we weren't getting the form response